### PR TITLE
Certificates refactor to consume the new API

### PR DESCRIPTION
**Pull request by javivelasco**

### CODE REVIEW

File: src/providers/sh/commands/certs.js [hunk 1]
```diff
@@ -5,20 +5,20 @@ const path = require('path')
...
-const mri = require('mri')
 const fs = require('fs-extra')
+const mri = require('mri')
 const ms = require('ms')
 const plural = require('pluralize')
-const printf = require('printf')
 const psl = require('psl')
-const supportsColor = require('supports-color')
+const table = require('text-table')
+require('epipebomb')()
```
Comment (rauchg) [inline src/providers/sh/commands/certs.js]: -1

File: src/providers/sh/commands/certs.js [hunk 2]
```diff
@@ -5,20 +5,20 @@ const path = require('path')
 
 // Packages
 const chalk = require('chalk')
-const table = require('text-table')
-const mri = require('mri')
 const fs = require('fs-extra')
+const mri = require('mri')
```
Comment (rauchg) [inline src/providers/sh/commands/certs.js]: `arg`

File: src/providers/sh/commands/certs.js [hunk 3]
```diff
@@ -92,308 +89,283 @@ const main = async ctx => {
...
+  })
+}
+
+async function deleteCertById(now, id) {
+  return now.fetch(`/v3/now/certs/${id}`, {
+    method: 'DELETE',
+  })
+}
+
+function sortByCn(certsList) {
```
Comment (rauchg) [inline src/providers/sh/commands/certs.js]: this needs comments, not obvious

File: src/providers/sh/commands/certs.js [hunk 4]
```diff
@@ -57,29 +55,28 @@ const help = () => {
...
-`)
+  `)
 }
 
-// Options
-let argv
-let debug
-let apiUrl
-let subcommand
+module.exports = async function main(ctx) {
```
Comment (rauchg) [inline src/providers/sh/commands/certs.js]: set this type to `: Promise<number>`

**Comment by OlliV** [comment]
Maybe you could use this for table formatting: https://github.com/zeit/now-cli/pull/1203

**Review by rauchg — APPROVED** [review]

**Pull request merged**